TAF_EasyRest/TestFramework/Tools/DBConnectionWrapper.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Npgsql;
+
+namespace TestFramework.Tools
+{
+    internal class DBConnectionWrapper
+    {
+
+        private static readonly string pg_connectionString = "Host=localhost;Username=admin;Password=12345678;Database=easyrest";
+
+        static void ShowHeaders(NpgsqlDataReader reader)
+        {
+            var columnNames = Enumerable.Range(0, reader.FieldCount)
+                        .Select(reader.GetName)
+                        .ToArray();
+            foreach (var column in columnNames) { Console.Write($"{column}\t"); }
+            Console.WriteLine();
+        }
+
+        static void ShowRows(NpgsqlDataReader reader)
+        {
+            while (reader.Read())
+            {
+                var row = Enumerable.Range(0, reader.FieldCount)
+                    .Select(reader.GetValue).ToArray();
+                foreach (var column in row) { Console.Write($"{column}\t"); }
+                Console.WriteLine();
+            }
+        }
+
+        public static void ExecuteQuery(string queryString)
```

Comment:
how do you plan to handle the result of the query execution in the test?

Reply:
I plan to query some changes in the database (e.g. DELETE FROM users WHERE email = 'kellymeza@test.com') and use the result in the TearDown - so that's how I can clear my DB after the test.

Reply:
how do you want to handle the result if the method returns `void`?